armi/tests/tutorials/c5g7-settings.yaml

```diff
@@ -2,6 +2,7 @@ metadata:
   version: uncontrolled
 settings:
   availabilityFactor: 0.9
+  fakeBad: 123456
```

Comment:
Could you post the output from the stdout when this is added? Just want to confirm that I understand the warning that is made.

Reply:
Sure. The total output is, of course, hundreds of lines. But luckily, this part is right at the top:

```
[warn] Ignoring invalid settings: fakeBad
[warn] Burn chain already imposed. Skipping reimposition.
=========== Case Information ===========
[info] ---------------------  ---------------------------------------------------
       Case Title:            c5g7-settings
       Case Description:      C5G7 LWR Benchmark inputs
       Run Type:              Standard - Operator
       Current User:          john
       ARMI Location:         /home/john/git/terrapower/armi/armi
       Working Directory:     /home/john/git/terrapower/armi/armi/tests/tutorials
       Python Interpreter:    3.8.10 (default, Sep 28 2021, 16:10:42)
                              [GCC 9.3.0]
       Master Machine:        ?
       Number of Processors:  1
       Date and Time:         Wed Dec  1 06:20:31 2021
...
```